src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/PersonsTest.kt

```diff
@@ -0,0 +1,84 @@
+package fi.metatavu.timebank.api.test.functional.tests
+
+import fi.metatavu.timebank.api.test.functional.TestBuilder
+import fi.metatavu.timebank.api.test.functional.data.TestData
+import fi.metatavu.timebank.api.test.functional.resources.LocalTestProfile
+import fi.metatavu.timebank.api.test.functional.resources.TestWiremockResource
+import fi.metatavu.timebank.api.test.functional.resources.TestMySQLResource
+import io.quarkus.test.common.QuarkusTestResource
+import io.quarkus.test.junit.QuarkusTest
+import io.quarkus.test.junit.TestProfile
+import org.junit.jupiter.api.*
+import org.junit.jupiter.api.Assertions.*
+
+/**
+ * Tests for Person API
+ */
+@QuarkusTest
+@QuarkusTestResource.List(
+    QuarkusTestResource(TestMySQLResource::class),
+    QuarkusTestResource(TestWiremockResource::class)
+)
+@TestProfile(LocalTestProfile::class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class PersonsTest: AbstractTest() {
+
+    /**
+     * Resets Wiremock scenario states before each test
+     */
+    @BeforeEach
+    fun resetScenariosBeforeEach() {
+        resetScenarios()
+    }
+
+    /**
+     * Tests /v1/persons?active=false -endpoint
+     */
+     @Test
+    fun listActivePersons() {
```

Comment:
Fix this in all methods
```suggestion
    @Test
    fun listActivePersons() {
```